### ISSUE TITLE
Question about the parameters for train and eval

### ISSUE THREAD

**Issue by Rolandxx7** [happinesslz/EPNetV2#1]
Hi, @happinesslz 

Thanks for your great job!

I have two questions for your work: 
1. I have found that the parameter called "MC_MASK_THRES" in your pretrained model is different from that in [run_train_and_eval_epnet_plus_plus_car.sh](https://github.com/happinesslz/EPNetV2/blob/main/tools/run_train_and_eval_epnet_plus_plus_car.sh). Could you please explain the reason, and which one is more suitble?
2. You said you would transfer EPNet++ to Waymo dataset, but the number of RGB images of Waymo dataset is different from the KITTI dataset. Waymo has multiple RGB images as input, so how are you going to deal with this problem?

**Comment by happinesslz** [comment]
@Rolandxx7 
Thanks for your attention. 
1.) MC_MASK_THRES is used in MC loss, you could refer to the formula(4) in the original paper of sec3.3. When the proproblities of belonging to the foreground for a position on two modalities are lower than a specified threshold, we think it is a background point. We do not compute MC loss for these background points. In general, it is proper to set MC_MASK_THRES as 0.2 in a dense scene~(64-beam). For a sparser scene, it is necessary to reduce the value MC_MASK_THRES~(e.g., 0.1 or 0.5).  
2.) Waymo provides 360-degree LiDAR points and multiple RGB images. And the point-wise correspondence between two modalities can be obtained by the provided multiple projection matrices.

**Comment by Rolandxx7** [comment]
@happinesslz 
Thanks for your detailed explanation！I have one more question to ask.

"PC_AREA_SCOPE" and "RPN_POST_NMS_TOP_N" defined in [PED_EPNet_plus_plus.yaml](https://github.com/happinesslz/EPNetV2/blob/main/tools/cfgs/PED_EPNet_plus_plus.yaml) are different from those in [CYC_EPNet_plus_plus.yaml](https://github.com/happinesslz/EPNetV2/blob/main/tools/cfgs/CYC_EPNet_plus_plus.yaml) and [CAR_EPNet_plus_plus.yaml](https://github.com/happinesslz/EPNetV2/blob/main/tools/cfgs/CAR_EPNet_plus_plus.yaml). Could you please explain the reason for me?

Also, I find that if we use the [[-40, 40], [-0.5,  2.5], [0, 70.4]] as the PC_AREA_SCOPE in [PED_EPNet_plus_plus.yaml](https://github.com/happinesslz/EPNetV2/blob/main/tools/cfgs/PED_EPNet_plus_plus.yaml) , the number of objects can be increased, so I would like to know why you did this.

**Comment by happinesslz** [comment]
@Rolandxx7 
For Pedestrian, we find setting a small range like offical [PointPillars](https://github.com/nutonomy/second.pytorch/blob/449c7c0d081eaad44f08159f64af26d2a59f1f4c/second/configs/pointpillars/ped_cycle/xyres_16.proto#L4) can promise a   satisfying performance. Besides, increasing the value of RPN_POST_NMS_TOP_N can improve the recall, which is very important to detection hard Pedestrians.

**Comment by Rolandxx7** [comment]
@happinesslz 
I'm really sorry for opening the issue again.
I am confused by these parameters in [PED_EPNet_plus_plus.yaml](https://github.com/happinesslz/EPNetV2/blob/main/tools/cfgs/PED_EPNet_plus_plus.yaml): 

What's the difference between "SCORE_THRESH" of RCNN, "NMS_THRESH" of RCNN , and "RPN_NMS_THRESH" of TEST?
